### PR TITLE
Fature: Pasword.

### PR DESCRIPTION
**Pull request by Far-Davari**

### CODE REVIEW

**Comment by Far-Davari** [comment]
Hi, I made a little change.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Giraffe
 about Giraffes!
 Here is a sample line
+It feels good!
```
Comment (Farshad2000) [inline README.md]: It's okay,  but if You Capitalize "f" in "feels" it would be great!

**Review by Farshad2000 — CHANGES_REQUESTED** [review]
read the suggestion.

File: README.md (hunk 1 above)
Comment (Far-Davari) [reply README.md]: okay, done!

**Pull request merged**